documentation/src/components/component.ts

```diff
@@ -27,7 +27,7 @@ enum TabValue {
 }
 
 interface JsDocTagParsed {
```

Comment:
I'm curious of why we're making the `tag` property optional? 
I also can't find where the property is being used 🤔 What does `tag` represent?